templates/vue/src/App.vue

```diff
@@ -70,7 +70,9 @@ export default {
         })
       })
     }
-
+    Promise.resolve(client.getAvatar()).then(savedAvatar => {
+      this.addAvatar(savedAvatar.data)
+    })
```

Comment:
Would this still occurs with no user is logged in?

Reply:
That code still runs even if the user is not logged in. I think we should prevent it from running if that is the case, but I think Aidin mentioned something about a mechanism forcing users to log in on the TYDE app.